src/Playlist.tsx

```diff
@@ -33,10 +37,21 @@ export const PlayList: React.FC = () => {
     return <div>Playlist Loading from {endpoint}</div>;
   }
 
+  if (itemClicked !== -1) {
+    return <Songlist playlistId={data.items[itemClicked].id}></Songlist>;
```

Comment:
I know this isn't super high pri, but want to make a note that this is better handled by `react-router` rather than returning the songlist here.